mods/d2k/rules/defaults.yaml

```diff
@@ -446,18 +446,22 @@
 	Inherits: ^Building
 	Inherits@selection: ^SelectableCombatBuilding
 	WithSpriteTurret:
+		RequiresCondition: !build-incomplete
 	AttackTurreted:
 	RenderRangeCircle:
 	DetectCloaked:
 		Range: 1c768
 	-GivesBuildableArea:
-	-WithMakeAnimation:
-	-WithCrumbleOverlay:
+	WithMakeAnimation:
+		BodyNames: make
 	-Capturable:
-	-WithSpriteBody:
+	WithSpriteBody:
+		Name: make
+		Sequence: null
```

Comment:
Can you explain why this is needed?

Reply:
I need this for the `make` animation, but not afterwards. Couldn't do this with conditions alone.

Reply:
I mean the `Sequence: null` specifically.